src/main/java/uk/gov/hmcts/reform/notificationservice/config/QueueClientConfig.java

```diff
@@ -4,34 +4,42 @@
 import com.microsoft.azure.servicebus.IMessageReceiver;
 import com.microsoft.azure.servicebus.ReceiveMode;
 import com.microsoft.azure.servicebus.primitives.ServiceBusException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 
+import java.util.Arrays;
 
 @Configuration
 public class QueueClientConfig {
 
+    private static final Logger log = LoggerFactory.getLogger(QueueClientConfig.class);
+
+    private static String ENTITY_PATH = "EntityPath=";
+
     @Bean
     @ConditionalOnProperty(name = "queue.notifications.read-connection-string")
     public IMessageReceiver notificationsMessageReceiver(
         @Value("${queue.notifications.read-connection-string}") String connectionString)
         throws InterruptedException, ServiceBusException {
+        logQueueName(connectionString);
         return ClientFactory.createMessageReceiverFromConnectionString(
             connectionString,
             ReceiveMode.PEEKLOCK
         );
     }
 
-    @Bean
-    @ConditionalOnProperty(name = "queue.notifications.staging-enabled")
-    public IMessageReceiver notificationsTestMessageReceiver(
-        @Value("${queue.notifications.staging-read-connection-string}") String connectionString)
-        throws InterruptedException, ServiceBusException {
-        return ClientFactory.createMessageReceiverFromConnectionString(
-            connectionString,
-            ReceiveMode.PEEKLOCK
-        );
+    private void logQueueName(String connectionString) {
+        String[] split = connectionString.split(";");
+        Arrays.stream(split)
+            .filter(e -> e.startsWith(ENTITY_PATH))
+            .findFirst()
+            .ifPresent(c ->
+                log.warn("Notification service connected to queue: {} ", c.replace(ENTITY_PATH, ""))
```

Comment:
Why 'warn'?

Reply:
as it is important and it will print once , Warning might be better.

Reply:
Yes - have seen Don's comment. Still not sure 'warn' is right - anyway we will search for 'Notification service connected to queue'. 'warn' means something not quite right.

Reply:
We should worry about the queue as if there is miss configuration application can work on wrong queue. I think we should "warn" the user about the queue.

Reply:
I believe this should be asked in the channel

Reply:
you can check the queue name on app boot

Reply:
check against what?

Reply:
in container startup logs